x/launchpad/types/msg_create_sale.go

```diff
@@ -44,7 +44,7 @@ func (msg *MsgCreateSale) validate() (sdk.AccAddress, []string) {
 	if msg.TokenOut == "" {
 		errmsgs = append(errmsgs, "`token_out` must be not empty")
 	}
-	if msg.InitialDeposit.GT(sdk.NewInt(d)) {
+	if !msg.InitialDeposit.GT(sdk.NewInt(d)) {
```

Comment:
This is wrong, we shouldn't compare to `d`. I've changed it in fd1cab1cbc223c5454fe171a971659ba134665f1